contracts/utils/contracts/src/SafeMath.sol

```diff
@@ -53,31 +53,15 @@ contract SafeMath {
         return c;
     }
 
-    function max64(uint64 a, uint64 b)
```

Comment:
not handy?

Reply:
We don't actually use `uint64` anywhere in our codebase AFAIK, not sure why this was there in the first place.